src/components/AssetTimeseriesPanel/stories/helper.tsx

```diff
@@ -0,0 +1,74 @@
+import { GetTimeSeriesMetadataDTO } from '@cognite/sdk';
+import { CogniteAsyncIterator } from '@cognite/sdk/dist/src/autoPagination';
+import React from 'react';
+import { sleep } from '../../../mocks';
+import { timeseriesListV2 } from '../../../mocks/';
+import { ClientSDKProvider } from '../../ClientSDKProvider';
+import {
+  MockDatapointsClientObject,
+  MockTimeseriesClientObject,
+  TimeseriesMockClient,
+} from '../../TimeseriesChart/stories/TimeseriesChart.stories';
+
+class CogniteClient extends TimeseriesMockClient {
+  timeseries: any = {
+    ...MockTimeseriesClientObject,
+    list: (): CogniteAsyncIterator<GetTimeSeriesMetadataDTO[]> => ({
+      // @ts-ignore TODO - remove this ts-ignore after fixing SDK
```

Comment:
think this comment can be removed

Reply:
Still, an error shows if remove this comment

Reply:
strange
ok, leave it as is for now